src/audio/src/src.c

```diff
@@ -323,6 +335,49 @@ int src_polyphase_init(struct polyphase_src *src, struct src_param *p,
 	return n_stages;
 }
 
+static int src_check_buffer_sizes(struct comp_dev *dev, struct comp_data *cd,
+				  struct audio_stream __sparse_cache *source_stream,
+				  struct audio_stream __sparse_cache *sink_stream)
```

Comment:
please keep this function where it was if possible

Reply:
Because the location of the code referring to the "src_check_buffer_sizes" function has changed, the location needs to be changed